week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,9 @@
 */
 
 var startingValue = 5;
+//var result = (format(startingValue*2)+10);
+var result = format(multiply(startingValue, 2) + add(0, 10));
```

Comment:
Pretty good! The only thing that I would change is that you shouldn't do "+ add(0, 10)" to add 10, it's a bit redundant to do it that way. Instead you should do this add(multiply(startingValue, 2), 10) so that the add function actually takes the multiplied value as its first parameter and 10 as its second parameter.